commons/all/src/main/java/org/infinispan/commons/util/IntSet.java

```diff
@@ -127,17 +126,13 @@ default void forEach(IntConsumer action) {
     * <p>The {@code Spliterator.OfInt} reports {@link Spliterator#DISTINCT}.
     * Implementations should document the reporting of additional
     * characteristic values.
-    *
-    * @implSpec
```

Comment:
Any reason we are removing these? Should the previous line just be moved up? It is standard for implSpec to talk about the default implementation.

Reply:
implSpect, apiNote and implNote are not quite standard: they are not processed automatically by JavaDoc and would require custom tag configs